src/core/context/info-cache-file.ts

```diff
@@ -0,0 +1,19 @@
+import { asArray, asObject, asString } from 'cleaners'
+
+import { EdgePluginMap, JsonObject } from '../../browser'
+import { asJsonObject, makeJsonFile } from '../../util/file-helpers'
+
+export interface InfoCacheFile {
+  corePlugins?: EdgePluginMap<JsonObject>
+  syncServers?: string[]
+}
+
+export const INFO_CACHE_FILE_NAME = 'infoCache.json'
+
+export const asInfoCacheFile = asObject<InfoCacheFile>({
+  // date: asDate,
```

Comment:
I had considered storing the date we fetched this, but I guess it ended up not being needed - I should probably delete this.